README.md

```diff
@@ -112,43 +172,6 @@ This list is for websites, services, software, tools and more: everything that y
 - [GC-Prevue](http://www.graphicode.com/GC-Prevue_Gerber_Viewer) - Commercial with free version. Can handle some gerbers better than Gerbv and KiCAD.
 - [ZofZPCB](https://www.zofzpcb.com/) - FREE 3D Gerber Viewer.
 
-## Free EDA Packages
-- [KiCad](https://kicad.org/) - Open source EDA package with push and shove router, differential pairs and much more.
-- [Eagle](https://www.autodesk.com/products/eagle/overview) - One of the most popular EDA packages due to it's (board size restricted) free version.
-- [DesignSpark PCB](https://www.rs-online.com/designspark/pcb-software) - Gratis EDA package without restrictions, sponserd by RS Components.
-- [Altium CircuitMaker](https://circuitmaker.com/) - Free package from the maker of the go to pro software.
-- [gEDA](http://geda-project.org) - Another open source package, good for people that like scripting and makefiles, Linux and BSD only.
-- [DipTrace](https://diptrace.com) - Quality Schematic Capture and PCB Design software with (pin and signal layer restricted) free version.
-- [LibrePCB](https://librepcb.org/) - A new, powerful and intuitive EDA tool for everyone, cross-platform and GNU GPLv3.
-- [Horizon EDA](https://github.com/horizon-eda/horizon) - A free and open source EDA tool with the focus on shortcut operation.
```

Comment:
I don't beleive Hozion EDA is mature enough to be put a an awesome item. It is not even close to Kicad, Proteus, Altium, and so on... Even as an alternative to Kicad, LibrePCB is much more suitable. If you have a deep knowledge in the horizon EDA and disagree, let me know.

Reply:
Maybe add a "tier" column, instead of deleting it?

Reply:
@joaoantoniocardoso I found that a great idea! Indeed, the list we have so far ranges from full-featured professional EDAs geared to complex electronics systems to simple hobbits singe-featured software. Maybe creating a tier category can help us to better distinguish them.

Reply:
I will work on this tier list as well as other info in `contributing.md`.

Reply:
Sorry to say: I think this tier thing is too complicated for this list. If we want to keep the unmaintained EDAs lets add a column for "maintenance status" and the other columns are for feature comparison so...